gitensite/apps/bookinfo/db.py

```diff
@@ -14,6 +20,22 @@ def addBookFromYaml(yaml):
 
     if "_repo" in obj:
         book.repo_name = obj["_repo"]
+        if "covers" in obj:
+            num_existing_covers = Cover.objects.filter(book=book).count()
+            for cover in obj["covers"]:
+                #Upload cover to S3
+                url = urlparse.urljoin("https://raw.githubusercontent.com/GITenberg/" + obj["_repo"] + "/master/", cover["image_path"])
+                r = requests.get(url)
+                contentfile = ContentFile(r.content)
+                uploadpath = obj["_repo"] + ".png"
+
+                #Add cover to database
+                coverobject = Cover.objects.create(
```

Comment:
if the cover has been previously loaded, does this create a duplicate?

Reply:
No. I specifically tested this and it overwrites the original file, which I believe is the behavior we want.

Reply:
ok